client/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "client",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "keywords": [],
+  "description": "",
+  "dependencies": {
+    "components": "^0.1.0",
```

Comment:
Let's delete this one. I think parcel added this automatically when we tried `components/Dummy` without `./` in front.